beacon-chain/sync/subscriber_test.go

```diff
@@ -40,6 +57,38 @@ func TestSubscribe_ReceivesValidMessage(t *testing.T) {
 	}
 }
 
+func TestSubscribe_WaitToSync(t *testing.T) {
+	p2p := p2ptest.NewTestP2P(t)
+	r := RegularSync{
+		ctx: context.Background(),
+		p2p: p2p,
+	}
+
+	r.chain = &mockChain.ChainService{}
```

Comment:
if you need to get the test to work you can modify `mockChain.ChainService{}` definition. Allow it to define a state initialized feed and then use that everytime
```go
// ChainService defines the mock interface for testing
type ChainService struct {
	State               *pb.BeaconState
	Root                []byte
	FinalizedCheckPoint *ethpb.Checkpoint
	StateFeed           *event.Feed
}

// StateInitializedFeed mocks the same method in the chain service.
func (ms *ChainService) StateInitializedFeed() *event.Feed {
	if ms.StateFeed != nil {
		return ms.StateFeed
	}
	return new(event.Feed)
}
```

Reply:
checking if it doesn't break other tests